ivy/functional/backends/tensorflow/elementwise.py

```diff
@@ -812,18 +792,7 @@ def erf(
     return tf.math.erf(x)
 
 
-@with_unsupported_dtypes(
-    {
-        "2.13.0 and below": (
-            "uint8",
-            "uint16",
-            "uint32",
-            "uint64",
-            "complex",
-        )
-    },
-    backend_version,
-)
+@with_unsupported_dtypes({"2.13.0 and below": ("complex", "bool")}, backend_version)
```

Comment:
have you tested uints for this function?

Reply:
yep:
![image](https://github.com/unifyai/ivy/assets/100947451/fcc08e42-5de6-4c6b-a0ef-89aa36310c3e)

Reply:
awesome, thanks for clarifying!